packages/ui-docs-client/src/Hero/index.js

```diff
@@ -285,6 +285,17 @@ class Hero extends Component {
             </Link>
           </List.Item>
         </List>
+        <Heading as="h3" level="h2" margin="none none medium">
```

Comment:
This block is cool. Should we add other badges? So lonely now, and we do use other products (e.g. Chromatic?)